guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc

```diff
@@ -53,6 +53,8 @@ euid = __ID_of_Apache_User__
 ----
 . Create a keytab entry:
 +
+WARNING: While creating a keytab entry, a host should not be connected to a domain.
```

Comment:
```suggestion
WARNING: Do not connect the host to a domain before creating a keytab entry.
```

Reply:
Better :+1:

Reply:
Waiting for technical review and even waiting to know if it is absolutely required.

Reply:
I think the more correct term is "join" or "enroll", so maybe "The host should not be enrolled to a domain before creating…"?

That said my understanding of AD domains is very limited and I can not really say something about the change (it *sounds* plausible, but that's all).